hail/src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -382,7 +382,8 @@ class IRSuite extends HailSuite {
   }
 
   @Test def testComplexInferPType() {
-    var ir = ArrayMap(
+    // InferPType expects array->stream lowered ir
+    val ir = LowerArrayToStream(ArrayMap(
```

Comment:
would prefer to rewrite teh IR to be inferrable here -- ToArray on the outside and change the ArrayRange to a StreamRange